consensus/common_test.go

```diff
@@ -14,6 +14,8 @@ import (
 
 	"github.com/go-kit/kit/log/term"
 
+	"path"
```

Comment:
```suggestion
	"filepath"
```

path does not work on Windows